### ISSUE TITLE
The website doesn't work

### ISSUE THREAD

**Issue by turion** [lambdacms/lambdacms.org#1]
When I go on http://lambdacms.org/, there is just a standard page saying:

```
Welcome to nginx!

If you see this page, the nginx web server is successfully installed and working. Further configuration is required.
```
And so on. Since I don't know any other way of contacting the web devs, I'm posting this as an issue here, although it's not an issue with the code of the website.

**Comment by geraldus** [comment]
Yes, unfortunately this is known issue and website currently is turned off.  If I understood correctly, Lambda CMS currently is unmaintained.  I was given access to project though and I have some plans on it.  In nearest future I plan to remove some redundant code and implement possibility of resource based access (these changes are in development already).

If I will have real use case (and I hope I will) it is possible to host demo site somewhere else and also publish some video content about look and feel of CMS.

Hope this helps.

**Comment by geraldus** [comment]
Also, @cies can clarify if it's possible to bring website up

**Comment by davidak** [comment]
The domain is also not available anymore. This project looks dead.

ERR_NAME_NOT_RESOLVED

For historic research, you can find the last version here http://web.archive.org/web/20160622014637/http://lambdacms.org/
